package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@netdata/netdata-ui",
-  "version": "1.9.17",
+  "version": "1.10.0",
```

Comment:
Do we need to do a minor update instead of patching?

Reply:
I think a minor update fits here because this is an addition to the iconList API (although a small one), and not a bugfix.

Reply:
Agreed on that, but it's a tiny change for doing a `minor` update!
Approved by me, up to you! 😛